python/tests/lis/test_loading.py

```diff
@@ -0,0 +1,41 @@
+"""
+Testing general loading functionality. For more information on tests refer to
+same tests in dlis section
+"""
+
+import pytest
+
+import shutil
+import os
+
+from dlisio import lis
+
+def test_filehandles_closed(tmpdir):
+    # Copy the test file to a tmpdir in order to make this test reliable.
+    tmp = str(tmpdir.join('file'))
+    shutil.copyfile('data/lis/layouts/layout_tif_01.lis', tmp)
+
+    with lis.load(tmp) as files:
+        assert len(files) == 4
+
+    os.remove(tmp)
+
+def test_filehandles_closed_when_load_fails(tmpdir, assert_error):
+    # TODO: add test files for every step that can fail, as in dlis
+
+    tmp = str(tmpdir.join('file'))
+    shutil.copyfile('data/lis/layouts/truncated_15.lis', tmp)
+
+    # file is truncated, but 2 LFs was already processed successfully
+    with lis.load(tmp) as files:
+        assert len(files) == 2
+    # TODO: change for explicit raise here as assert_error is nor reliable
```

Comment:
It would be nicer to have 
```
with pytest.raises
      lis.load(tmp)
```

if we ever introduce errorhandler and throw.
Throwing is the thing that actually is important for this test as it's supposed to delete all the filehandlers. Error-handled situation is to be dealt with in "non-failing" test.
By not reliable I meant that to make it more reliable one needs to use overload assert_message_count (or what it is called now) or create different test case for every failing function, which should come in future.

Basically that's a TODO for an ErrorHandler change, if it ever happens :D 
Can change the note if you want to though :)